project.clj

```diff
@@ -3,7 +3,12 @@
   :url "https://github.com/RickMoynihan/lein-tools-deps"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/tools.deps.alpha "0.3.260"]]
+  :dependencies [[org.clojure/tools.deps.alpha "0.5.417"]
+                 [org.clojure/tools.logging "0.2.3"]
+                 [ch.qos.logback/logback-classic "1.0.13"]
```

Comment:
Is there a reason we need tools.logging and logback?  I was trying to keep dependencies to a minimum.  Also IIRC leiningen plugins should use the leiningen logger

> When emitting output, please use leiningen.core.main/info, leiningen.core.main/warn, and leiningen.core.main/debug rather than println since these will respect the user's output settings.

[From: Writing a Plugin](https://github.com/technomancy/leiningen/blob/master/doc/PLUGINS.md#writing-a-plugin)

Reply:
I needed logging to understand what was going on. Are you wanting it removed or replaced with lien logging?

Reply:
I think it should be replaced with lein logging to remove the extra dependencies, which we should try and keep to a minimum to prevent any unexpected conflicts with other plugins, user code etc.

I can't remember quite how the leiningen logging stuff works, IIRC it's pretty basic; if it's noisy for normal users then you may just want to leave the line commented out.